images/win/scripts/Installers/Update-AndroidSDK.ps1

```diff
@@ -90,9 +112,16 @@ if (Test-Path $ndkRoot) {
     setx ANDROID_NDK_ROOT $ndkRoot /M
     (Get-Content -Encoding UTF8 "${ndkRoot}\ndk-build.cmd").replace('%~dp0\build\ndk-build.cmd','"%~dp0\build\ndk-build.cmd"')|Set-Content -Encoding UTF8 "${ndkRoot}\ndk-build.cmd"
 } else {
-    Write-Host "NDK is not installed at path $ndk_root"
+    Write-Host "LTS NDK $ndkLTSMajorVersion is not installed at path $ndkRoot"
     exit 1
 }
 
+$ndkLatestPath = "$sdkRoot\ndk\$($ndkLatestFullVersion.Split(';')[1])"
```

Comment:
rewrite blocks if to check the path only:

```
if (-not (Test-Path $ndkRoot)) {
	Write-Host "LTS NDK $ndkLTSMajorVersion is not installed at path $ndkRoot"
    exit 1
}

$ndkLatestPath = "$sdkRoot\ndk\$($ndkLatestFullVersion.Split(';')[1])"
if (-not (Test-Path $ndkLatestPath)) {
	Write-Host "Latest NDK $ndkLatestMajorVersion is not installed at path $ndkLatestPath"
    exit 1
}
```

Reply:
We set environments variables with these paths if these paths exist, not only check.